packages/flutter_localizations/lib/src/l10n/cupertino_en.arb

```diff
@@ -1,12 +1,42 @@
 {
+  "@datePickerHourSemanticsLabelZero": {
```

Comment:
I _think_ that this should be okay with the internal Google translation console scripts, but just commenting on this since I'm not exactly sure what having a resource attribute without a matching resource id is a normal pattern. But then again, the Material and Cupertino localizations handles plurals in an unusual way, so this could be okay.

Reply:
Yeah, it was the simplest solution I could think of for this. If we do have problems with the Google translation console, we can try to come up with another way.